src/core/BufferGeometry.js

```diff
@@ -645,8 +655,17 @@ BufferGeometry.prototype = Object.assign( Object.create( EventDispatcher.prototy
 					var morphAttribute = morphAttributesPosition[ i ];
 					_boxMorphTargets.setFromBufferAttribute( morphAttribute );
 
-					_box.expandByPoint( _boxMorphTargets.min );
-					_box.expandByPoint( _boxMorphTargets.max );
+					if ( this.morphTargetsRelative ) {
+
+						_box.boundingBox.expandByVector( _boxMorphTargets.min );
+						_box.boundingBox.expandByVector( _boxMorphTargets.max );
```

Comment:
Should be `_box.expandByVector(` here I think.

Reply:
Yup - just noticed this as well when testing decals. expandByVector also doesn't do quite what I thought it's doing so I think this is slightly incorrect - I'll double check and fix this.